build.py

```diff
@@ -0,0 +1,64 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
+
+import os
+import sys
+import logging
+import subprocess
+
+YELLOW_FG = "\x1b[33m"
+DEFAULT_FG = "\x1b[39m"
+
+log = logging.getLogger("build")
+logging.basicConfig(
+    level=logging.DEBUG, format=f"{YELLOW_FG}[%(levelname)s] %(message)s{DEFAULT_FG}"
+)
+
+PYTHON_PATH = sys.executable
+log.debug(f"PYTHON_PATH={PYTHON_PATH}")
+
+
+def run(cmd, **kwargs):
+    log.debug(f">>> {cmd}")
+    ret = subprocess.run(cmd, shell=True, check=True, **kwargs)
+    return ret
+
+
+def in_cibuildwheel():
+    return bool(int(os.environ.get("CIBUILDWHEEL", "0")))
+
+
+def check_venv():
+    # Both conda and venv are configured, pick conda
+    if os.environ.get("CONDA_PREFIX") and os.environ.get("VIRTUAL_ENV"):
+        del os.environ["VIRTUAL_ENV"]
+    # Build with cibuildwheel detected, set `VIRTUAL_ENV`
+    if in_cibuildwheel():
```

Comment:
Maybe change it to:
```suggestion
    if in_cibuildwheel() and not os.environ.get("VIRTUAL_ENV"):
```
So we don't override a configured venv, even when cibuildwheel is used?

Reply:
done in 5996000